config.json

```diff
@@ -887,6 +887,12 @@
       ]
     },
     {
+      "uuid": "8fe44264-0eb3-b780-635b-1ac0df69d7ddbc2d68c",
+      "slug": "complex-numbers",
+      "difficulty": 3,
+      "topics": [ "Mathematics" ]
+    },
```

Comment:
Please add new exercises to the end of the list of problems.

Reply:
OK.
I thought the right place would be above the two deprecated ones.

Reply:
Oh. it might be. 

Sorry, I clearly didn't look closely enough at where you'd put them.